### DISCUSSION TITLE
SEB 2.3.2 MacOS crash after 1s open

### DISCUSSION THREAD

**Discussion by namxhung** [SafeExamBrowser/seb-mac#119] [Q&A]
I have a problem with SEB, help me!!

SEB version 2.3.2 running on my Macbook crashes and shows the log ([https://pastebin.com/raw/8tCSBYbT](https://pastebin.com/raw/8tCSBYbT)) after about 1s opening. I can't do my test, can anyone help me?

I tried deleting and reinstalling but still the same
![image](https://user-images.githubusercontent.com/43926774/131217341-5913daf7-75d4-4549-8f37-ec44e4511cdb.png)

**Comment by danschlet** [comment]
That's a crash in the WebKit web browser engine, not in the SEB code itself. Probably some incompatibility. Maybe the [new version SEB 3.0](https://github.com/SafeExamBrowser/seb-mac/releases/tag/3.0b1) will solve it, if your exam web application meets the requirements so SEB 3.0 can use the modern WebView (which has better compatibility with modern web APIs and if the web rendering process crashes, SEB isn't terminated as well).

You could also try to start your Mac in [Safe Mode](https://support.apple.com/en-us/HT201262) when using SEB, in case it's some third-party tool which causes the crash.
